### ISSUE TITLE
No dirty tracking methods when using only `store_attribute`

### ISSUE THREAD

**Issue by gsmetal** [palkan/store_attribute#11]
If I use 
```ruby
store_accessor :settings, some_time: :datetime
```
there is `saved_change_to_some_time?` (and other dirty tracking methods), but if 

```ruby
store_attribute :settings, :some_time, :datetime
```
there is no.

**Comment by palkan** [comment]
Good catch. Thanks!
Will fix soon

For now you can try the following hack:

```ruby
# this would add dirty methods
store_accessor :settings, :some_time
# this would add type casting
store_attribute :settings, :some_time, :datetime
```

(No 100% sure though)

**Comment by palkan** [comment]
Released in 0.8.1
